### PR TITLE
[Android/SingleShot] Add `Options` class for configuring SingleShot

### PR DESCRIPTION
**Pull request by anyj0527**
- Add Options to configure SingleShot instance preparing new features that might be added in future
- Add three negative test cases for newly added `Options`

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

File: api/android/api/src/main/java/org/nnsuite/nnstreamer/SingleShot.java [hunk 1]
```diff
@@ -337,4 +343,116 @@ public void close() {
...
+         * @param type              The type of {@link NNStreamer.NNFWType}
+         * @param models            The array of {@link File} objects to the neural network model files
+         * @param inputInfo         The input tensors information {@link TensorsInfo}
+         * @param outputInfo        The output tensors information {@link TensorsInfo}
+         * @param custom            The custom option string to open the neural network instance
+         *
+         * @throws IllegalArgumentException if given models is invalid
+         * @throws IllegalStateException    if given framework is not available
+         */
+        private Options(NNStreamer.NNFWType type, File[] models, TensorsInfo inputInfo, TensorsInfo outputInfo, String custom) {
```
Comment (jaeyun-jung) [inline api/android/api/src/main/java/org/nnsuite/nnstreamer/SingleShot.java:397]: let's make this public.

Comment (anyj0527) [reply api/android/api/src/main/java/org/nnsuite/nnstreamer/SingleShot.java]: Updated!

**Review by again4you — APPROVED** [review]
LGTM!

**Pull request merged**